src/components/MovieInfo/MovieInfo.js

```diff
@@ -23,28 +23,31 @@ class MovieInfo extends Component {
   render() {
 
     return (
-      this.state.currentMovie ?
-      <section className='movie-card'>
-        <img src={this.state.currentMovie.backdrop_path} alt='movie backdrop' className='movie-backdrop' />
-        <div className='movie-display'>
-          <img src={this.state.currentMovie.poster_path} alt='movie poster' className='movie-poster' />
-          <article className='movie-stats'>
-            <h2>{this.state.currentMovie.title}</h2>
-            <h3>{this.state.currentMovie.average_rating}</h3>
-            <p>{this.state.currentMovie.release_date}</p>
-            <p>{this.state.currentMovie.overview}</p>
-            <p>{this.state.currentMovie.genres}</p>
-            <p>{this.state.currentMovie.budget}</p>
-            <p>{this.state.currentMovie.revenue}</p>
-            <p>{this.state.currentMovie.runtime}</p>
-            <p>{this.state.currentMovie.tagline}</p>
-            <Link to='/'>
-              <button>Return Home</button>
-            </Link>
-          </article>
-        </div>
-      </section> :
-      <h1>Loading...</h1>
+      <>
```

Comment:
Glad we got this rendering logic figured out in the AllMovies component--the App component looks much better